Sources/Castor/Cast/Cast.swift

```diff
@@ -190,6 +190,9 @@ extension Cast: @preconcurrency GCKSessionManagerListener {
             resume(from: resumeState)
             delegate?.castStartSession()
         }
+        else {
```

Comment:
Nice catch. 

To make it even more explicit that the start session delegate method must be called, I would suggest we use an if/else instead of an if/elseif/else and just handle the resume optionally within the else branch:

```swift
if let resumeState = targetResumeState {
   // ...
}
else {
    if let resumeState = castable?.castStartSession() {
        resume(from: resumeState) 
    }
    delegate?.castStartSession()
}
```